docs/content/3.manual/manual.yml

```diff
@@ -408,15 +412,16 @@ sections:
           Like `.[]`, but no errors will be output if . is not an array
           or object.
 
-      - title: "`,`"
+      - title: "Comma, Output Stream Join: `,`"
```

Comment:
Calling it the "comma" operator is... concise, but not precise.  I can't think of a one-word precise name for it.  "Output Stream" is not correct since `.[]` too (and others) creates an output stream, for example.

The opcodes don't always provide good names for things.  `.[]` is clearly a value iterator, but internally it's called `EACH`.  I would rather call `.[]` an iterator than an "each".

The actual semantics of `,` is that the whole thing outputs the outputs of the expression on the left, then those of the expression on the right.

Perhaps "Expression Concat"?  Something like that.

I must say I don't like "Output Stream Join" either, though mostly because it will seem to a new user like an odd, off-putting name for what should obviously be a trivial thing.  It's too soon here to make it clear that jq is all about generators, and that generators are pervasive.